python/taichi/lang/__init__.py

```diff
@@ -483,6 +483,8 @@ def check_version():
         payload['python'] = 'cp38'
     elif python_version.startswith('3.9.'):
         payload['python'] = 'cp39'
+    elif python_version.startswith('3.10.'):
+        payload['python'] = 'cp310'
```

Comment:
Can be changed to a more concise form:
```python
python_version = sys.version_info
payload['python'] = f'cp{python_version[0]}{python_version[1]}'
```

Reply:
thanks, i believe you mean this?
```
python_version = platform.python_version().split('.')
payload['python'] = f'cp{python_version[0]}{python_version[1]}'
```

Reply:
`sys.version_info` is a tuple of version bits so we don't need to split it ourselves.

Reply:
i see, thanks a lot, changed!